tests/test_basis.py

```diff
@@ -1395,7 +1569,7 @@ def test_evaluate_returns_expected_number_of_basis(
                 f"The first dimension of the evaluated basis is {eval_basis.shape[1]}",
             )
 
-    @pytest.mark.parametrize("sample_size", [6, 30, 35])
+    @pytest.mark.parametrize("sample_size", [12, 30, 35])
```

Comment:
why this change?

Reply:
6 samples is not enough for the frequencies that i am testing for. It raises the aliasing error in the Fourier basis init